.github/workflows/tests.yml

```diff
@@ -21,15 +21,18 @@ jobs:
         uses: actions/checkout@v4
          
       - name: Setup Conda/Mamba
-        uses: conda-incubator/setup-miniconda@v2
+        uses: conda-incubator/setup-miniconda@v3
         with:
-          mamba-version: "*"
+          mamba-version: "2"
+          channel-priority: true
+          auto-update-conda: false
           activate-environment: mirflowz
           environment-file: environment.yml
+          conda-remove-defaults: true
           auto-activate-base: false
```

Comment:
I think it would make sense to always specify the arguments in the same order whenever you use the `setup-miniconda` action. If it doesn't break anything (which I guess/hope it shouldn't), perhaps the best is to use increasing alphabetic order.